mrdo_arcade.asm

```diff
@@ -10550,7 +10707,7 @@ cvb_ANIMATEDLOGO:
 
 	CALL 	MYDISSCR
 	
-	LD		HL, $1900				; remove sprites in the new position of the SAT
+	LD		HL, PNT				; remove sprites in the new position of the SAT

```

Comment:
This should be SAT (or stay $1900)!!! Me bad sorry!

Reply:
No worries. I can fix when I merge!